docs/config.md

```diff
@@ -205,3 +167,11 @@ The explanation and default value of each configuration item are as follows:
       [plugins.cri.registry.mirrors."docker.io"]
         endpoint = ["https://registry-1.docker.io", ]
 ```
+
```

Comment:
If using an untrusted-workload pod annotation to request a pod be run using a runtime for untrusted workloads, the RuntimeHandler `plugins.cri.containerd.runtimes.untrusted` must be defined first. When the annotation `io.kubernetes.cri.untrusted-workload` is set to `true` the `untrusted` runtime will be used. For example, see [Create an untrusted pod using Kata Containers(https://github.com/kata-containers/documentation/blob/master/how-to/how-to-use-k8s-with-cri-containerd-and-kata.md#create-an-untrusted-pod-using-kata-containers).

Reply:
Done